docs/gestor_tareas.md

```diff
@@ -0,0 +1,13 @@
+# Gestor de tareas
+Para el proyecto se necesitará un gestor de tareas que se usará para automatizar la ejecución de tests y el despliegue.
+
+Se han considerado las herramientas Invoke y Poetry.
+
+Invoke es un gestor de tareas que puede llevar a cabo funciones de gestión de dependecias. Se utiliza para la automatización de la construcción de proyectos además de poder realizar otras tareas como la automatización de test.
```

Comment:
No, no puede, salvo que llame a un gestor de dependencias. Se trata de que entendáis claramente qué función tiene cada uno.